client/layout/index.jsx

```diff
@@ -94,7 +94,8 @@ Layout = React.createClass( {
 			<MasterbarLoggedIn
 				user={ this.props.user }
 				section={ this.props.section }
-				sites={ this.props.sites } />
+				sites={ this.props.sites }
+				isSupportUser={ config.isEnabled( 'support-user' ) && this.props.isSupportUser } />
```

Comment:
I wouldn't pass this as a prop. I'd set a class on `layout` and use that as `.layout .masterbar {}`